src/core/agent/services/multiSigService.ts

```diff
@@ -831,7 +839,11 @@ class MultiSigService extends AgentService {
         recp,
         { gid: aid }
       );
+      authorizedEids.push(eid);
     }
+    await this.identifierStorage.updateIdentifierMetadata(hab["prefix"], {
+      authorizedEids,
+    });
```

Comment:
This commit is kind of conflicting with something you said elsewhere.

Every wallet in the multi-sig will try to do the end role auth once the multi-sig operation is done, so every wallet will have all 3 eids in a 2 of 3 multi-sig in this field.

I think you said before that the wallets still need to call join on the authorisation before the OOBI is ready. But now every wallet will ignore the messages from the other wallets since the eid is already in the array and never join. Am I misunderstanding?

Reply:
What I mean is every wallet will need to be the one that triggers the authorization or the one that joins the authorization.

Reply:
Hmm, but this was my original point. So technically, all wallets could call `endRoleAuthorization` and we could completely remove any joining.

Anyway, I think we still need to handle it like this because the `eids` can change over time. (which we haven't covered yet, but another wallet we interact with might)

Cool